### PR TITLE
fix(interactive): hide opposite theme code blocks

### PR DESCRIPTION
**Pull request by zhangzheheng12345**

### CODE REVIEW

File: interactive/components/Summary.vue [hunk 1]
```diff
@@ -10,8 +13,13 @@ const isDefault = $computed(() => (userConfigRaw.value || defaultConfigRaw) ===
...
         {{ searcher.uno.config.presets.length }}
       </div>
-      <div op50>
+      <div op50 @mouseenter="showPresets = true" @mouseleave="showPresets = false">
         Presets
+        <div v-if="showPresets" ml-2>
+          <div v-for="item in searcher.uno.config.presets" :key="item.name">
+            {{ item.name }}
+          </div>
+        </div>
```
Comment (zyyv) [inline interactive/components/Summary.vue:22]: This change is irrelevant, if you want see all presets, you can switch config panel.

**Comment by zhangzheheng12345** [comment]
emm yeah I think so, but the idea came to me when I first entered the document website and want to know what presets are loaded. Perhaps that's my silliness.

So I have to revert these things?

File: interactive/components/Summary.vue (hunk 1 above)
Comment (zhangzheheng12345) [reply interactive/components/Summary.vue:22]: emm yeah I think so, but the idea came to me when I first entered the document website and want to know what presets are loaded. Perhaps that's my silliness.

So I have to revert these things?

File: interactive/components/details/Guide.vue [hunk 2]
```diff
@@ -17,7 +19,7 @@ watch(
 </script>
 
 <template>
-  <DetailsBase v-if="component" :title="item.title">
+  <DetailsBase v-if="component" :title="item.title" :class="{ light: !isDark, dark: isDark }">
```
Comment (zyyv) [inline interactive/components/details/Guide.vue:22]:
```suggestion
  <DetailsBase v-if="component" :title="item.title" :class="isDark ? 'dark' : 'light'">
```

**Comment by zhangzheheng12345** [comment]
OK I've changed that.

**Review by zyyv — APPROVED** [review]

**Pull request merged**